tests/MakesAssertionsTest.php

```diff
@@ -487,33 +570,6 @@ public function test_assert_not_focused()
         }
     }
 
-    public function test_assert_selected()
```

Comment:
- Renamed this test to `test_assert_selected_and_element_is_selected()` and moved it further up the page to attempt to follow the method order as seen in `MakeAssertions.php`